src/Files.App/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml.cs

```diff
@@ -63,7 +63,7 @@ private void HorizontalTabView_TabItemsChanged(TabView sender, Windows.Foundatio
 		private async void TabViewItem_Drop(object sender, DragEventArgs e)
 		{
 			await ((sender as TabViewItem).DataContext as TabItem).Control.TabItemContent.TabItemDrop(sender, e);
-			HorizontalTabView.CanReorderTabs = true;
+			HorizontalTabView.CanReorderTabs = true && !ElevationHelpers.IsAppRunAsAdmin();
```

Comment:
```suggestion
			HorizontalTabView.CanReorderTabs = !ElevationHelpers.IsAppRunAsAdmin();
```

I think we can remove `true &&`, it's redundant. But we might consider leaving it there for when we will re-enable drag & drop as admin

Reply:
You're quite right, it's redundant!

Reply:
There are other occurrences in the same file, can you fix them too